.github/workflows/web-test-unit-tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Unit tests for web code
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/web-test.yml"
+      - "web-admin/**"
+      - "web-auth/**"
+      - "web-common/**"
+      - "web-local/**"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Filter modified codepaths
+        uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            admin:
+              - ".github/workflows/web-test.yml"
+              - "admin/**"
+              - "cli/**"
+              - "runtime/**"
+              - "web-admin/**"
+            auth:
+              - ".github/workflows/web-test.yml"
+              - "web-auth/**"
+            local:
+              - ".github/workflows/web-test.yml"
+              - "cli/**"
+              - "runtime/**"
+              - "web-local/**"
+            common:
+              - ".github/workflows/web-test.yml"
+              - "cli/**"
+              - "runtime/**"
+              - "web-common/**"
+
+      - name: Set up NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: NPM Install
+        run: npm install
+
+      - name: unit test web-common
+        if: steps.filter.outputs.common == 'true'
+        run: npm run test -w web-common
+
+      - name: Test web-auth
+        if: steps.filter.outputs.auth == 'true'
+        run: npm run test -w web-auth
+
+      - name: Make sure storybook is working
```

Comment:
This name threw me off a little bit. What about "Run storybook smoketest"

Reply:
And to have symmetry, the other names could be "Run web-common unit tests" and "Run web-auth unit tests"